.claude/commands/reviewsuper.md

```
@@ -16,6 +16,7 @@ execution_mode: immediate
 1. Analyze code for architectural issues
 2. Identify problems, flaws, and technical debt
 3. Provide specific, actionable recommendations
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)